component/custom/custom_loader.go

```diff
@@ -0,0 +1,40 @@
+package custom
+
+import (
+	"os"
+	"plugin"
+	"strings"
+)
+
+func init() {
+	pluginPath := os.Getenv("PLUGIN_PATH")
+	if pluginPath == "" {
+		return
+	}
+
+	// environment variable can contain multiple paths separated by colon.
+	paths := strings.Split(pluginPath, ":")
+
+	for _, file := range paths {
+		println("custom_loader::Opening plugin:", file)
```

Comment:
log.Info ?

Reply:
we don't have logger initialised at the time of init() :(

Reply:
using logger instead of println but the logs in init will go to syslog instead of the log file used.